test/e2e_node/services/services.go

```diff
@@ -64,15 +64,19 @@ func NewE2EServices(monitorParent bool) *E2EServices {
 // standard kubelet launcher)
 func (e *E2EServices) Start() error {
 	var err error
+	if e.services, err = e.startInternalServices(); err != nil {
+		return fmt.Errorf("failed to start internal services: %v", err)
+	}
+	klog.Infof("Node services started.")
 	if !framework.TestContext.NodeConformance {
-		if e.services, err = e.startInternalServices(); err != nil {
-			return fmt.Errorf("failed to start internal services: %v", err)
-		}
```

Comment:
I'm not entirely sure if it matters if services are started before or after the kubelet.